public/partials/codereview.html

```diff
@@ -1,7 +1,52 @@
-<div ng-controller="CodeCtr as code" id="code-canvas" draw knock>
```

Comment:
Why did you remove the Code Controller? This should be used by other applications